challenges/intermediate-generic-with-constraints/solution.py

```diff
@@ -0,0 +1,22 @@
+"""
+TODO:
+
+Define a function that accepts two parameters of the same type and returns the same type.
+The type can only be str or int.
+"""
+from typing import TypeVar
+
+T = TypeVar("T", int, str)
+
+def add(a, b: T) -> T:
```

Comment:
```suggestion
def add(a: T, b: T) -> T:
```

Otherwise the test doesn't pass

Reply:
done